datafusion/core/tests/sqllogictests/test_files/window.slt

```diff
@@ -2365,3 +2365,625 @@ SELECT c9, rn1 FROM (SELECT c9,
+                CsvExec: file_groups={1 group: [[WORKSPACE_ROOT/datafusion/core/tests/data/window_1.csv]]}, projection=[ts, inc_col, desc_col], output_ordering=[ts@0 ASC NULLS LAST], has_header=true
```

Comment:
It is neat you can see here what the output ordering is (so it is clearer that the test has sorted inputs)